### ISSUE TITLE
[Question]: UIE微调问题

### ISSUE THREAD

**Issue by yjt-okkk** [PaddlePaddle/PaddleNLP#4044]
### 请提出你的问题

使用UIE finetune.py，执行完do_train后直接跳出，不会继续执行do_eval与do_export

**Comment by linjieccc** [comment]
@yjt-okkk Hi,

能否提供一下你的执行配置

**Comment by yjt-okkk** [comment]
> @yjt-okkk Hi,
> 
> 能否提供一下你的执行配置
python 3.7.12
paddle 2.4.0

python finetune.py     -
-device gpu    
--logging_steps 10     
--save_steps 100    
--eval_steps 100     
--seed 42     
--model_name_or_path uie-base     
--output_dir ./checkpoint/model_best  
--train_path ./data/train.txt   
--dev_path ./data/dev.txt     
--max_seq_length 512    
--per_device_eval_batch_size 16   
--per_device_train_batch_size  16  
--num_train_epochs 100  
--learning_rate 1e-5   
--label_names 'start_positions' 'end_positions'    
--do_train True  
--do_eval True 
--do_export True   
--export_model_dir ./checkpoint/model_best  
--overwrite_output_dir  
--disable_tqdm True     
--metric_for_best_model eval_f1     
--load_best_model_at_end  True    
--save_total_limit 1

**Comment by linjieccc** [comment]
@yjt-okkk 请问是直接退出还是？有没有报错信息

**Comment by yjt-okkk** [comment]
> @yjt-okkk 请问是直接退出还是？有没有报错信息

直接退出，没有报错信息，如果不运行do_train，单独运行do_eval与do_export可以成功

**Comment by NLPOR** [comment]
同样的问题+1

**Comment by linjieccc** [comment]
windows环境么，可以观察下机器的显存/内存使用情况
